test/commands/scanner/run.test.ts

```diff
@@ -621,4 +600,77 @@ describe('scanner run', function () {
 
 		});
 	});
+
+	describe('Create internal outfile with SCANNER_INTERNAL_OUTFILE environment variable', () => {
```

Comment:
Added these three tests as end to end tests for now. These do add time to our test suite... so I hope that soon we can start pushing out end to end tests by one step and start implementing tests on the Action classes themselves soon to gain our speed back. But I didn't want to start that effort here in this work item.